src/ProjectWindow.cpp

```diff
@@ -677,7 +690,19 @@ ProjectWindow::ProjectWindow(wxWindow * parent, wxWindowID id,
          }
       });
 
-   wxTheApp->Bind(EVT_THEME_CHANGE, &ProjectWindow::OnThemeChange, this);
+   mThemeChangeSubscription =
+      theTheme.Subscribe(*this, &ProjectWindow::OnThemeChange);
+
+   mFocusChangeSubscription = TrackFocus::Get(project)
+      .Subscribe([this](const TrackFocusChangeMessage& msg) {
+         if(mEffectsWindow->IsShown())
+         {
+            auto& project = GetProject();
+            auto& trackFocus = TrackFocus::Get(project);
+            ShowEffectsPanel(project, trackFocus.Get());
```

Comment:
Will that line un-collapse the effects panel if it was hidden, whenever I use the up and down arrow keys?  Maybe not desirable if so.